src/main/java/io/kestra/plugin/nats/kv/CreateBucket.java

```diff
@@ -0,0 +1,175 @@
+package io.kestra.plugin.nats.kv;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.nats.NatsConnection;
+import io.nats.client.Connection;
+import io.nats.client.KeyValueManagement;
+import io.nats.client.api.KeyValueConfiguration;
+import io.nats.client.api.KeyValueStatus;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Creates Key/Value bucket in NATS."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Creates a new Key/Value bucket, with all required properties",
+            code = {
+                "url: nats://localhost:4222",
+                "username: nats_user",
+                "password: nats_passwd",
+                "name: my_bucket"
+            }
+        ),
+	    @Example(
+            title = "Creates a new Key/Value bucket",
+            code = {
+                "url: nats://localhost:4222",
+                "username: nats_user",
+                "password: nats_passwd",
+                "name: my_bucket",
+                "description: my bucket for special purposes",
+                "historyPerKey: 2",
+                "bucketSize: 1024",
+                "valueSize: 1024",
+                "metadata: {\"key1\":\"value1\",\"key2\":\"value2\"}"
+            }
```

Comment:
Just a side note, you can leave it like this but instead of using multiple strings for an example you can leverage Java text blocks, it allows to add double-quote in it so it's more easier to use.

For ex:
```java
code = {
                """
                url: nats://localhost:4222
                username: nats_user
                password: nats_passwd
                name: my_bucket
                description: my bucket for special purposes
                historyPerKey: 2
                bucketSize: 1024
                valueSize: 1024
                metadata: {"key1":"value1","key2":"value2"}"""
}
```